src/LVMGenerator.h

```diff
@@ -439,7 +441,7 @@ class LVMGenerator : protected RamVisitor<void, size_t> {
 
             size_t address_L0 = code->size();
 
-            code->push_back(LVM_ITER_NotAtEnd);  // Start the formal for loop if the relation is non-empty
+            code->push_back(LVM_ITER_NotAtEnd);  // Start the aggregate for loop
```

Comment:
If this comment is helpful, move it to the line before. We don't use in-line comments unless there is a really clear readability benefit, e.g., a series of lines of short code + short comment